pkg/token/execCredentialPlugin_test.go

```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/golang/mock/gomock"
 )
 
-func TestExecCredentialPlugin(t *testing.T) {
+func TestExecCredentialWriter(t *testing.T) {
```

Comment:
why is this change needed?

Reply:
@everjing, this file is to test execCredentialPlugin, why is this function changed?

Reply:
You are right, this change is not needed and this func name will be reverted in next commit. Thanks!